CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 Please check our [developers guide](https://gitlab.com/tokend/developers-guide)
 for further information about branching and tagging conventions.
+## [Unreleased]
+#### Added
+- Shown only small balances
+- Renamed russian translation
```

Comment:
Why in block `added` you `renamed`? What has been added? Use block `fixed` text: russian localization

Reply:
changed